.github/workflows/pr_review.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
         with:
           debug: true
-          review_simple_changes: false
-          review_comment_lgtm: false
+#          review_simple_changes: false
+#          review_comment_lgtm: false
```

Comment:
The line 'review_comment_lgtm: false' is commented out, which means it is not being used. This might have been done by mistake.